client/src/containers/UploadForm.js

```diff
@@ -32,6 +38,44 @@ class UploadForm extends Component {
     }
   }
 
+  uploadFile(files) {
+    const file = files[0];
+    const cloudName = keys.CLOUD_NAME;
+    const url = `https://api.cloudinary.com/v1_1/${cloudName}/image/upload`;
+    const timestamp = Date.now() / 1000;
+    const upload_preset = keys.IMAGE_UPLOAD_PRESET;
+    const api_secret = keys.IMAGE_API_SECRET;
+    const api_key = keys.IMAGE_API_KEY;
+    const paramStr = `timestamp=${timestamp}&upload_preset=${upload_preset}${
+      api_secret
+    }`;
+    const signature = sha1(paramStr);
+    const params = {
+      api_key,
+      timestamp,
+      upload_preset,
+      signature,
+    };
+
+    let uploadRequest = superagent.post(url);
+    uploadRequest.attach('file', file);
+
+    Object.keys(params).forEach(key => {
+      uploadRequest.field(key, params[key]);
+    });
+
+    uploadRequest.end((err, res) => {
+      if (err) {
+        alert(err);
+        return;
+      }
+
+      this.setState({ imageUrl: res.body.url });
+      console.log('Upload Successful', JSON.stringify(res.body.url));
+      console.log('State:', this.state);
```

Comment:
logs in the frontend are not great :sweat_smile:

Reply:
@Karyum took out console logs and cleaned up the config files!